api/src/main/java/javax/enterprise/inject/spi/builder/AnnnotatedConfiguratorsTest.java

```diff
@@ -0,0 +1,88 @@
+package javax.enterprise.inject.spi.builder;
+
+import javax.enterprise.event.Observes;
+import javax.enterprise.inject.spi.ProcessAnnotatedType;
+import javax.enterprise.util.AnnotationLiteral;
+import javax.inject.Inject;
+
+public class AnnnotatedConfiguratorsTest {
```

Comment:
Remove this class. It was intended just to demonstrate the usage of the `AnnotatedTypeConfigurator` API.